test/rpc/test_rpc_transactions.py

```diff
@@ -7,7 +7,7 @@
 from typing import List
 
 from starkware.starknet.definitions import constants
-from starknet_devnet.blueprints.rpc import RpcContractClass, RpcInvokeTransaction
```

Comment:
After removing this import, is `RpcInvokeTransaction` used anywhere at all?

Reply:
It is used in the rpc.py blueprint, for example in the `rpc_invoke_transaction()` function.

Reply:
Ah right, it's used for type specification.